hermes/probe/delete/delete.go

```diff
@@ -35,7 +35,9 @@ import (
 )
 
 const (
-	apiLatency = "hermes_api_latency_seconds" // TODO(evanSpendlove) add this constant to metrics.go
+	apiLatency             = "hermes_api_latency_seconds" // TODO(evanSpendlove) add this constant to metrics.go
+	minFileIDToDelete      = 11                           // we can delete files starting from the file Hermes_11
+	numberOfDeletableFiles = 40                           // there are 40 files to delete from [Hermes_11,Hermes_50]
```

Comment:
Nit: why not maxFileIDToDelete for symmetry?

Reply:
That's a good idea, I've changed it accordingly.